xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/widgets/locationPicker.js

```diff
@@ -276,7 +276,7 @@ require(['jquery', 'xwiki-meta', 'xwiki-events-bridge', 'xwiki-form-validation-a
         'reference': XWiki.Model.serialize(documentReference)
       }, function(data) {
         // Update the space reference part of the new location.
-        var newLocationContainer = $(data);
+        var newLocationContainer = $(data).find("ol.breadcrumb");
```

Comment:
Is this change still correct for non-navigation location pickers?

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2133/commits/0e679c1789c8626d7a747ef0f835fa35ca978cb7 :+1: